src/assets/scss/modules/_grid-system.scss

```
@@ -69,11 +69,22 @@
     margin-left: -(variables.$dp-spaces--lv2);
   }
 
+  .dp-colflex {
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    width: 100%;
+  }
+
   [class*="col-"] {
     position: relative;
     width: 100%;
     padding-right: variables.$dp-spaces--lv2;
     padding-left: variables.$dp-spaces--lv2;
+
+    &.dp-no-padding-col {
```

```suggestion
    p-all-0 {
```
No estoy muy seguro del nombre, no se si tenemos algo asi

---

este lo busque ` p-all-0` y no hay nada con ese nombre, asi que podria ser